examples/python/helloworld/helloworld_pb2_grpc.py

```diff
@@ -67,4 +99,41 @@ def SayHello(request,
             helloworld__pb2.HelloRequest.SerializeToString,
             helloworld__pb2.HelloReply.FromString,
             options, channel_credentials,
-            insecure, call_credentials, compression, wait_for_ready, timeout, metadata)
+            insecure, call_credentials, compression, wait_for_ready,
+            timeout, metadata, _registered_method=True)
+
+    @staticmethod
+    def SayHelloStreamReply(request,
+            target,
+            options=(),
+            channel_credentials=None,
+            call_credentials=None,
+            insecure=False,
+            compression=None,
+            wait_for_ready=None,
+            timeout=None,
+            metadata=None):
+        return grpc.experimental.unary_stream(request, target, '/helloworld.Greeter/SayHelloStreamReply',
```

Comment:
Can we put each argument on its own line? While we can't _guarantee_ that the lines won't exceed 80 characters because they contain user-provided strings, there's a much lower chance of this happening if each argument has its own line.

We don't provide a guarantee of any particular formatting protocol with our generated code, but we do have plenty of people who complain when we break common rules, such as 80 characters in length. The result is that people often either exclude `*pb2*.py` from their formatter's configuration or they manually format their files after generation.